Directory.Build.props

```diff
@@ -71,11 +71,10 @@
 
     <DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
   </PropertyGroup>
-  
-  <Import Project="$(RepositoryEngineeringDir)Analyzers.props" />
 
+  <!-- Disable SDK supplied netanalyzers as we reference them from nugets instead -->
```

Comment:
why reference them as nuget packages? according to the global.json file this repository uses the .NET 6 SDK: https://github.com/dotnet/msbuild/blob/b827bf58c21f7a38770d786848c76f771d995a94/global.json#L6 which means these come along from free. the nuget package is much older and is not going to be updated.

Reply:
This PR is closed. Could you raise the issue in #7192?